polls/urls.py

```diff
@@ -0,0 +1,13 @@
+from polls import views
+from django.conf.urls import url
+
+urlpatterns = [
+    url(r'^$', views.IndexView.as_view(), name='index'),
+    url(r'^(?P<pk>[0-9]+)/$', views.DetailView.as_view(), name='detail'),
+    url(
+        r'^(?P<pk>[0-9]+)/results/$',
+        views.ResultsView.as_view(),
+        name='results'
+    ),
+    url(r'^(?P<question_id>[0-9]+)/vote/$', views.vote, name='vote'),
```

Comment:
It doesn't make a huge difference but sometimes using a `\d` (digit) as a placeholder in regex is easier than `[0-9]`:
```python
>>>> match = re.match(r'(?P<pk>\d+)/$', '1234/')
>>>> match.groupdict()
{'pk': '1234'}
>>>>
```